File: ImadokiMusicPlayer/app/src/main/java/com/example/imadokimusicplayer/MainActivity.java
package com.example.imadokimusicplayer;

import android.content.Intent;
import android.support.design.widget.FloatingActionButton;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageButton;
import android.widget.ImageView;
import android.widget.SeekBar;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    MediaBrowserCompat mBrowser;
    MediaControllerCompat mController;

    // UI関係の宣言
    TextView textView_title;
    TextView textView_position;
    TextView textView_duration;
    ImageButton button_prev;
    ImageButton button_next;
    FloatingActionButton button_play;
    ImageView imageView;
    SeekBar seekBar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // TODO::UIのセットアップ
        textView_title = findViewById(R.id.textView_title);


        // サービスを開始する
        // Activity破棄と同時にServiceも停止して良いならこれは不要
        startService(new Intent(this, MusicService.class));
    }
}

File: QiitaClient/app/src/main/java/com/example/qiitaclient/MainActivity.kt
package com.example.qiitaclient

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.ListView
import com.example.qiitaclient.model.Article
import com.example.qiitaclient.model.User
import com.example.qiitaclient.view.ArticleView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val listAdapter = ArticleListAdapter(applicationContext)
        listAdapter.articles = listOf(dummyArticle("Kotlin入門", "たろう"),
            dummyArticle("Java入門", "じろう"))

        val listView: ListView = findViewById(R.id.list_view) as ListView
        listView.adapter = listAdapter
    }

    private fun dummyArticle(title: String, userName: String): Article =
            Article(id = "",
                title = title,
                url = "http://kotlinlang.org/",
                user = User(id = "", name = userName, profileImageUrl = ""))
}
